plugins/modules/database/postgresql/postgresql_idx.py

```diff
@@ -438,9 +442,9 @@ def drop(self, schema, cascade=False, concurrent=True):
             query += ' CONCURRENTLY'
 
         if not schema:
```

Comment:
@andytom , yes, i haven't found any reasons why the parameter needs to be there:)
Thanks for pointing out!
Would you like to fix this?
(if yes, feel free also remove the line ```4. Use self.module instead of passing arguments to the methods whenever possible``` from the class doc strings (abadger explained me why it's not good to extract parameters inside modules than pass them explicitly)

Reply:
Yep, leave it to me. I noticed some other refactoring that I'd like to do in the `postgresql_ext` so I'll pop up a PR to fix that too later today.

Reply:
Ok